<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReporteTortuga</class>
 <widget class="QMainWindow" name="ReporteTortuga">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reportes de las tortugas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/Images/if_Turtle_204384.ico</normaloff>:/Images/if_Turtle_204384.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(57, 134, 137)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbMain">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>981</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Raleway</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);
background-color: rgb(255, 170, 255);
background-color: rgb(231, 231, 231);</string>
    </property>
    <property name="title">
     <string>Reporte de tortugas</string>
    </property>
    <widget class="QLabel" name="lbMunicipio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Municipio</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneZona">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>60</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbInvestigador">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Investigador</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbIdTortuga">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Id Tortuga</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbNombreTortuga">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Nombre tortuga</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbZona">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>60</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Zona</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneSubespecie">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>30</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneNombreTortuga">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneIdTortuga">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneEstado">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>90</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneInvestigador">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbSubespecie">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Subespecie</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbEstado">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>90</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Estado</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneMunicipio">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>281</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbInvestigador_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>CÃ³digo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneCodigo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneReport">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>711</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway Black</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbRegister">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>210</y>
       <width>221</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton#pbRegister {
background-color: rgb(120, 190, 151);
 }

 QPushButton#pbRegister:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
     </property>
     <property name="text">
      <string>   Registrar reportes</string>
     </property>
     <property name="icon">
      <iconset resource="images2.qrc">
       <normaloff>:/report_save.ico</normaloff>:/report_save.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbActive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>981</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);
background-color: rgb(255, 170, 255);
background-color: rgb(231, 231, 231);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pbActive">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton#pbActive {
background-color: rgb(120, 190, 151);
 }

 QPushButton#pbActive:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
     </property>
     <property name="text">
      <string>  Activar</string>
     </property>
     <property name="icon">
      <iconset resource="images2.qrc">
       <normaloff>:/report_add.ico</normaloff>:/report_add.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pbExit">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>10</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton#pbExit {
background-color: rgb(120, 190, 151);
 }

 QPushButton#pbExit:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
     </property>
     <property name="text">
      <string>    Salir</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/Images/door_out.ico</normaloff>:/Images/door_out.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
  <include location="images2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>ReporteTortuga</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>823</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
